src/move_basic.cpp

```diff
@@ -95,9 +95,14 @@ class MoveBasic {
     int rotationAttempts;
     double localizationLatency;
 
+    double abortTimeout;
```

Comment:
This should be a ros::Duration, we should avoid doing time comparisons using floats or doubles.

This does make it slightly awkward to make into a parameter unfortunately, you will need a abortTimeoutSecs variable to get the double out from nm.param, but then you should immediately make a ros::Duration out of it in  abortTimeout,